scripts/model-diagram.mmd

```diff
@@ -0,0 +1,12 @@
+erDiagram
```

Comment:
Hmm, now this lives in scripts/, which is a bit weird... maybe it should live in top-level?

Reply:
Unsure how it would look if we get more diagrams... But for now SGTM.